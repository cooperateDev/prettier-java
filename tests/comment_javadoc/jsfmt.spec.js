run_spec(__dirname, ["java"]);
