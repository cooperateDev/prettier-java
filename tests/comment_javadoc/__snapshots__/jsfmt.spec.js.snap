// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comment_javadoc.java 1`] = `
class EmptyComment {
  /***/
}

class MultiComments {
  /** Abc */
  /** XYZ */
  /** Something */
}

class MultiCommentsWithEmptyLines {
  /** Abc */

  /** XYZ */

  /** Something */
}

class MultiCommentsWithDeclarationsAfter {
  /** Abc */int i;
  
  /** XYZ */public void doSomething(int j) {
    System.out.println("do");
  }



  
  /** Something */
}

class MethodComment {
  public void doSomething1(int j) {
    /** Abc */
    System.out.println("do");



    /** XYZ */
    System.out.println("do");


    
  }

  public void doSomething2(int j) {



    /** Abc */
    System.out.println("do");



    /** XYZ */
    System.out.println("do");


    /** Something */ 
  }
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class EmptyComment {
    /***/
}

class MultiComments {
    /** Abc */

    /** XYZ */

    /** Something */
}

class MultiCommentsWithEmptyLines {
    /** Abc */

    /** XYZ */

    /** Something */
}

class MultiCommentsWithDeclarationsAfter {
    /** Abc */
    int i;

    /** XYZ */
    public void doSomething(int j) {
        System.out.println("do");
    }

    /** Something */
}

class MethodComment {

    public void doSomething1(int j) {
        /** Abc */

        System.out.println("do");

        /** XYZ */

        System.out.println("do");
    }

    public void doSomething2(int j) {

        /** Abc */

        System.out.println("do");

        /** XYZ */

        System.out.println("do");

        /** Something */
    }

}


`;
